debezium-server-voyager/debezium-server-voyagerexporter/src/main/java/io/debezium/server/ybexporter/YbExporterConsumer.java

```diff
@@ -239,6 +256,20 @@ public void handleBatch(List<ChangeEvent<Object, Object>> changeEvents, Debezium
         handleSnapshotOnlyComplete();
     }
 
+    private boolean checkIfEventNeedsToBeWritten(Record r) {
+        if (r.isUnsupported()) {
+            LOGGER.debug("Skipping unsupported record {}", r);
+            return false;
+        }
+        if (eventCache != null && sourceType.equals("oracle")) {
```

Comment:
You should consider adding PG and YB support in this same PR.

Reply:
Will do it in the upcoming commits